protoc-gen-twirp_ruby/main.go

```diff
@@ -70,20 +70,26 @@ func (g *generator) Generate(in *plugin.CodeGeneratorRequest) *plugin.CodeGenera
 func (g *generator) generateFile(file *descriptor.FileDescriptorProto) *plugin.CodeGeneratorResponse_File {
 	pkgName := pkgName(file)
 	g.P(`# Code generated by protoc-gen-twirp_ruby, DO NOT EDIT.`)
-	for _, service := range file.Service {
+	g.P(`require 'twirp'`)
+	g.P(``)
+	g.P(fmt.Sprintf("module %s", CamelCase(pkgName)))
+	for i, service := range file.Service {
 		serviceName := serviceName(service)
-		g.P(`require 'twirp'`)
-		g.P(``)
-		g.P(fmt.Sprintf("class %sService < Twirp::Service", serviceName))
-		g.P(fmt.Sprintf(`  PATH_PREFIX = "/twirp/%s.%s"`, pkgName, serviceName))
+		g.P(fmt.Sprintf("  class %sService < Twirp::Service", serviceName))
+		g.P(fmt.Sprintf(`    PATH_PREFIX = "/twirp/%s.%s"`, pkgName, serviceName))
 		for _, method := range service.GetMethod() {
 			methName := methodName(method)
 			inputName := methodInputName(method)
 			outputName := methodOutputName(method)
-			g.P(fmt.Sprintf("  rpc :%s, %s::%s, %s::%s", methName, CamelCase(pkgName), inputName, CamelCase(pkgName), outputName))
+			g.P(fmt.Sprintf("    rpc :%s, %s, %s, handler_method: :%s",
```

Comment:
the `rpc` DSL should use a String for the method name instead of a Symbol:
```ruby
"rpc \"%s\", %s, %s, handler_method: :%s"
```

This will allow for more esoteric method names to "just work", and make more clear that the routing is done based on string matching